tailwind.config.js

```diff
@@ -54,6 +54,9 @@ module.exports = {
     maxWidth: {
       1440: '1440px',
     },
+    maxHeight: {
+      52: '13rem',
```

Comment:
question (blocking): this seems arbitrary. Afaik @moritzmurauer and @Abrom8 agreed on using pixels here and if you want to use rem why are using 13 for 52?

Reply:
This is a setting from a future tailwind version. See https://tailwindcss.com/docs/max-height.
As soon as we upgrade to the new major version we can drop this configuration.